### PR TITLE
Final what's new in 1.0 changes

### PR DESCRIPTION
**Pull request by astrofrog**
One last small change before 1.0rc2

### CODE REVIEW

**Comment by astrofrog** [comment]
@eteq - how did you derive these values before:

```
Since v0.3, by the numbers:

819 issues have been closed
511 pull requests have been merged
57 distinct people have contributed code
```

Do you have a script? Can we add something similar here? Do we include v0.4.1 to v0.4.4 in the numbers?

**Comment by eteq** [comment]
@astrofrog - Honestly I don't remember.  I think I had some hacked together thing to grab the stats from the github API.  Something like https://github.com/astropy/astropy-tools/blob/master/gitastropyplots.py but with a check for when exactly it happened.

**Comment by embray** [comment]
Incidentally, I just ran
```
[embray@bray:astropy]$ python3.2 setup.py test
```
and got
```
Traceback (most recent call last):
  File "setup.py", line 45, in <module>
    cmdclassd = register_commands(NAME, VERSION, RELEASE)
  File "/internal/1/root/src/astropy/astropy/astropy_helpers/astropy_helpers/setup_helpers.py", line 297, in register_commands
    add_command_hooks(registered_commands, srcdir=srcdir)
  File "/internal/1/root/src/astropy/astropy/astropy_helpers/astropy_helpers/setup_helpers.py", line 330, in add_command_hooks
    for setuppkg in iter_setup_packages(srcdir, packages):
  File "/internal/1/root/src/astropy/astropy/astropy_helpers/astropy_helpers/setup_helpers.py", line 561, in iter_setup_packages
    name=packagename + '.setup_package')
  File "/internal/1/root/src/astropy/astropy/astropy_helpers/astropy_helpers/utils.py", line 243, in import_file
    loader = import_machinery.SourceFileLoader(name, filename)
AttributeError: 'module' object has no attribute 'SourceFileLoader'
```

Should astropy-helpers support Python 3.2?  I can't remember if there has been a decision on that point or not.  If it needs to it could.  If it doesn't need to it should still give a nicer error message instead of just blowing up at the first occasion something doesn't work.

File: docs/whatsnew/1.0.rst [hunk 1]
```diff
@@ -9,6 +9,24 @@ What's New in Astropy 1.0?
...
 
+Astropy 1.0 is a major release that adds significant new functionality since the 0.4.x series of releases.
+
+In particular, coordinate conversions to/from Altitude/Azimuth are now
+supported (see `Support for Alt/Az coordinates`_), a new package to help with
+data visualization has been added (see :ref:`whatsnew_viz`), and a new package
+for common analytic functions is now also included (see
+:ref:`whatsnew_analytical_functions`).
+
+The :mod:`astropy.io.ascii` sub-pacakge now includes fast C-based
```
Comment (embray) [inline docs/whatsnew/1.0.rst]: "pacakge"

File: docs/whatsnew/1.0.rst [hunk 2]
```diff
@@ -9,6 +9,24 @@ What's New in Astropy 1.0?
...
+readers/writers for common formats, and also supports a new ASCII format the
```
Comment (embray) [inline docs/whatsnew/1.0.rst]: the$ -> that$

File: docs/whatsnew/1.0.rst [hunk 3]
```diff
@@ -9,6 +9,24 @@ What's New in Astropy 1.0?
...
+better preserves meta-data (see :ref:`whatsnew_io_ascii`), the modeling package
+has been significantly improved and now supports composite models (see
+:ref:`whatsnew_modeling`), and the :class:`~astropy.table.Table` class can now
+include :class:`~astropy.coordinates.SkyCoord` and :class:`~astropy.time.Time`
+objects containing arrays (see :ref:`whatsnew_table`).
+
+In addition to these major changes, Astropy 1.0 includes a large number of
+smaller improvements, which are described in the `Full change log`_
```
Comment (embray) [inline docs/whatsnew/1.0.rst]: Maybe mention that there are bug fixes unique to this release as well. (Just "smaller improvements *and* bug fixes")

**Comment by astrofrog** [comment]
@embray - I implemented your comments, thanks!

Regarding Python 3.2, I think it's fine to not support it (I thought that by deciding to not support astropy for 3.1 or 3.2 that decision automatically extended to astropy-helpers). Are you thinking that we should raise an error like ``astropy-helpers is not supported on Python 3.2`` instead of the current error?

**Comment by eteq** [comment]
Looks good, so I'm going to go ahead and merge.  We can add the number of issues closed section just before release so it has the latest numbers.

A agree no need to support py 3.2 in the helpers if we're not going to support it in astropy (unless some affiliated package specifically needs it, and I can't imagine that's true right now).

**Pull request merged**